css/category.css

```diff
@@ -0,0 +1,54 @@
+.category-container {
+  width: 100vw;
+  min-height: 100vh;
+  margin: 0 auto;
+  background-color: var(--bg-dark);
+}
+
+.category-head {
+  margin: 0 auto;
+  text-align: center;
+  padding: 1rem 0;
+  font-family: "Palanquin Dark", sans-serif;
+}
+
+.category-list {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
can use utility classes for these.
Due to time constraints it can be refactored later

Reply:
Yes, will do that will next fix.